cassandra/roles/restore-cassandra/files/restore-backup.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: cassandra
+  namespace: velero
```

Comment:
Can we change this to a template so that the velero namespace can be changed? I have Velero currently installed in `oadp-operator` so I would like to pass this in as a var:
`$ ansible-playbook restore-cassandra.yml -e velero_namespace=oadp-operator`

Reply:
added variables in commit af11f648bcb5d41c53a2ac5cb9fe955b32608fbf to address this